recipes-connectivity/networkmanager/files/nm_to_db.awk

```diff
@@ -20,7 +20,8 @@ BEGIN {
     gsub(/'/, "'\\''", key);
     gsub(/'/, "'\\''", value);
 
-    system("db-write-dom0 '/" node "/" file "/" group "/" key "' '" value "'");
+    if (key != "permissions")
```

Comment:
```
if (key == "permissions" && group == "connection")
    continue
```
To be extra careful that it's in the expected group?

Reply:
yea I can add that